server/controllers/likeController.js

```diff
@@ -19,7 +19,7 @@ export default class LikeController {
     const dislike = req.params.likeType === 'dislike';
     // message to be sent to user depending on function performed
     const message = like || dislike ? `you ${req.params.likeType}d the article`
-      : 'you disliked the article';
+      : 'you unliked the article';
```

Comment:
I edited the like controller to fix a bug of not disliking an article after liking that particular article.